proto/cosmos/accesscontrol/constants.proto

```diff
@@ -37,5 +37,23 @@ enum ResourceType {
     KV_BANK_SUPPLY = 18; // child of KV
     KV_BANK_DENOM = 19; // child of KV
     KV_BANK_BALANCES = 20; // child of KV
+    KV_TOKENFACTORY_DENOM = 21; // child of KV_TOKENFACTORY
+    KV_TOKENFACTORY_METADATA = 22; // child of KV_TOKENFACTORY
+    KV_TOKENFACTORY_ADMIN = 23; // child of KV_TOKENFACTORY
+    KV_TOKENFACTORY_CREATOR = 24; // child of KV_TOKENFACTORY
+    KV_ORACLE_EXCHANGE_RATE = 25; // child of KV_ORACLE
```

Comment:
do we need to do anything to the existing oracle ones (do they conflict in any way)? It looks fine, but not sure if any changes we made affect them

Reply:
They should be fine, I tried to make sure that each prefix key has their own child, so at least it should be 1-1